src/main/java/trello/entities/List.java

```diff
@@ -84,7 +84,7 @@ public void setCards(final ArrayList<Card> cards) {
      *
      * @return an ArrayList with all cards in the list.
      */
-    public ArrayList<Card> getCards() {
+    public java.util.List<Card> getCards() {
```

Comment:
Please go up this import to the section of import.

Reply:
This is not an import. It is like that so the IDE couldn't confuse between the java.util.List and the entity List that we are using in the context.